django/core/cache/backends/memcached.py

```diff
@@ -202,3 +202,9 @@ def __init__(self, server, params):
     @cached_property
     def _cache(self):
         return self._lib.Client(self._servers, **self._options)
+
+    def close(self, **kwargs):
+        # Override base behavior of calling disconnect_all(), since this resets
+        # the failover state, and creates an unnecessary amount of reconnects.
+        # libmemcached manages its own connections, so isn't affected by #5133.
```

Comment:
I think ", so isn't affected by #5133." could be omitted. We try to reference tickets in code only for obscure issues that can't easily be described in comments.